src/javascript/ContentManager/ContentManager.jsx

```diff
@@ -38,7 +38,13 @@ class ContentManager extends React.Component {
 
         this.defaultNS = 'content-media-manager';
         this.namespaceResolvers = {
-            'content-media-manager': lang => require('../../main/resources/javascript/locales/' + lang + '.json')
+            'content-media-manager': lang => {
+                if (['fr', 'en', 'de'].indexOf(lang) >= 0) {
```

Comment:
This will prevent customers to provide custom translations. With this we are restricted to only the 3 languages we provide.

Reply:
This issue applies as well to the changes in other files within this pull request.